config/env_spec_request.go

```diff
@@ -440,10 +446,10 @@ func mustBeInClaim(value, name string, claims map[string]interface{}) error {
 	return fmt.Errorf("%q not in claim %q", value, name)
 }
 
-// IsAuthenticated returns true if AuthenticatationRequirements are met for the request
-// Returns true if AuthenticatationRequirements are empty or disabled.
-func (e *EnvironmentSpecRequest) IsAuthenticated() bool {
-	return e.meetsAuthenticatationRequirements(e.getAuthenticationRequirement())
+// Authenticate returns nil if AuthenticatationRequirements are met for the request
+// Returns nil if AuthenticatationRequirements are empty or disabled.
```

Comment:
Ditto here-- also,  you have two lines that basically say the same thing. Pick one? :)

Reply:
Rephrased.
The two statements are different. First says, nil if met and second says empty/disabled is also considered met.